README.md

```diff
@@ -12,7 +12,7 @@ It identifies the use of deprecated code thanks to the `@deprecated` annotation.
 
 ## Basic knowledge
 
-The Sensiolabs DeprecationDetector is a command line command based on the [Symfony2 console component](https://github.com/symfony/Console).
+The Sensiolabs DeprecationDetector is a command line command based on the [Symfony console component](https://github.com/symfony/Console).
```

Comment:
Should be "Symfony Console component" (notice the uppercased "C"). It's not related to your changes, but it would be nice if you could change it right away too while you are at it.